assemblies/lib/pom.xml

```diff
@@ -197,6 +202,61 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.rabbitmq</groupId>
+      <artifactId>amqp-client</artifactId>
+      <version>${amqp366.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
```

Comment:
Why does the 5.2 version need to be here, isn't it only needed by the osgi plugin?  There will be conflicts if both are in the app classpath.